Dockerfile

```diff
@@ -102,15 +136,14 @@ RUN jupyter nbextension enable --py widgetsnbextension --sys-prefix \
 ENV XDG_CACHE_HOME /home/$NB_USER/.cache/
 RUN MPLBACKEND=Agg python -c "import matplotlib.pyplot"
 
-USER root
-
-### Custom stuff 
 
+### Custom stuff
 # Install KBC Transformation package
-RUN pip3 install --no-cache-dir --upgrade git+git://github.com/keboola/python-transformation.git@1.1.13
+RUN pip3 install --no-cache-dir --upgrade --user git+git://github.com/keboola/python-transformation.git@1.1.13

```

Comment:
run that under root

Reply:
yes, it is run as root, the `--user` option is left-over from a previous attempt, I'll remove it, thanks